lib/config/presets/index.ts

```diff
@@ -266,7 +266,8 @@ export async function resolveConfigPresets(
   inputConfig: AllConfig,
   baseConfig?: RenovateConfig,
   _ignorePresets?: string[],
-  existingPresets: string[] = []
+  existingPresets: string[] = [],
+  shallowLog = false
```

Comment:
```suggestion
  existingPresets: string[] = [],
  shallowLog = true
```

Reply:
you can set the fallback value to `true` and then you dont need to change the calling line below.
that said, if you only call this function once or even few times in the same mode than the flag is redundant.

ive seen this function being called only once.